core/templates/dev/head/pages/footer_js_libs.html

```diff
@@ -1,4 +1,3 @@
-<script src="/templates/dev/head/AppInit.js"></script>
```

Comment:
Why this change? I didn't see this one in the original PR.

Reply:
This is important as now the AppInit.ts file is not transpiled to JS and hence cannot be imported as a JS file. It is used directly as a TS file.

Reply:
Ah, okay